test/unit/lib/email-service-spec.js

```diff
@@ -121,10 +140,18 @@ describe('Email Service', () => {
           });
         });
 
-        it('calls emailer.sendEmail passing to address, subject and rendered templates', done => {
-          emailService.sendEmail('sterling@archer.com').then(() => {
+        it('calls emailer.sendEmail passing to address, customer subject and rendered templates', done => {
+          emailService.sendEmail('sterling@archer.com', 'customer').then(() => {
+            emailService.emailer.sendEmail.should.have.been.calledOnce
+              .and.calledWith('sterling@archer.com', 'A customer email', ['html', 'html']);
+            done();
```

Comment:
FYI, you can just return to the `it` rather than explicitly call `done`.
```
it('....', () =>
   emailService.sendEmail('sterling@archer.com', 'customer').then(() => 
             emailService.emailer.sendEmail.should.have.been.calledOnce
               .and.calledWith('sterling@archer.com', 'A customer email', ['html', 'html']);
    )
);
```